### ISSUE TITLE
Centering issues again

### ISSUE THREAD

**Issue by VSEphpbb** [phpbb-extensions/ad-management#29]
@senky @Derky 

Since we added an inner div to our ad templates, we have centering issues again.

This time the issue affects users who want to add like an Advertisement heading to their ads.

![screen shot 2017-06-16 at 2 14 58 pm](https://user-images.githubusercontent.com/303711/27245819-8db07a32-52a2-11e7-8c99-c7f97f8c715a.png)

We can make our inner div have a width of 100%. But that introduces a new problem, because the flex-box is centering the inner div, which would now be full width, so any ad codes inside will be left justified again.

![screen shot 2017-06-16 at 2 51 26 pm](https://user-images.githubusercontent.com/303711/27245969-4d4f7fe6-52a3-11e7-9e8b-f20a2c7c4633.png)

This then forces the user to find their own way to center the ad in their ad code.

Basically what we need is full width support of the page (i.e.: the parent div the ad is in) and the ad needs to be centered, without applying any of its formatting to the internals of the ad itself.

**Comment by Derky** [comment]
Besides your fix we could also decide to just remove the text-alignment so the admin can add it's own CSS and is not forced to have it center.

**Comment by VSEphpbb** [comment]
I'd rather it be centered by default, and only make the user add extra css if they want to do something more than just the simple default behavior or centering.

**Comment by Derky** [comment]
Ok yeah I guess you can just do a float:left if you want to overwrite it.
